test/spec/controllers/groupCtrlSpec.js

```diff
@@ -23,51 +23,207 @@ describe('Groups Controller', function() {
+     *
+     * This coercion business is troublesome...
+     */
+    /*
+    describe("isMember behavior", function() { 
+      it("should return true dependent on whether the user id belongs to the group", function() {
+        expect(scope.isMember(user, guild)).to.eql(truthy);
+        expect(scope.isMember(user, party)).to.eql(falsy);
       });
+    });
+    */
+  });
 
-      var myGuilds = sandbox.stub(groups,"myGuilds", function() {
-        return [guild];
+  describe("Quest Membership Behavior", function() { 
+
+    var runningQuestParty, pendingQuestParty;
+
+    beforeEach(function() {
+      runningQuestParty= specHelper.newGroup({
+         _id: "unique-party-id",
+         type: 'party',
+         members: ['leader-id'],
+         quest: {
+           progress: {},
+           active: true,
+           members: {}
+         }
+      });
+      pendingQuestParty= specHelper.newGroup({
+         _id: "unique-party-id",
+         type: 'party',
+         members: ['leader-id'],
+         quest: {
+           progress: {},
+           active: false,
+           members: {}
+         }
       });
+      runningQuestParty.quest.members[user._id] = true;
+      pendingQuestParty.quest.members[user._id] = true;
+    });
+ 
+    it('should return false if no quest in the group', function() {
+      runningQuestParty.quest = undefined;
+      expect(scope.isMemberOfPendingQuest(user._id, runningQuestParty)).to.eql(false);
+      expect(scope.isMemberOfRunningQuest(user._id, runningQuestParty)).to.eql(false);
+    });
 
-      expect(scope.isMemberOfGroup(user._id, guild)).to.be.ok;
-      expect(myGuilds).to.be.called;
+    it('should return false if no members in the group quest', function() {
+      runningQuestParty.quest.members = {};
+      expect(scope.isMemberOfPendingQuest(user._id, runningQuestParty)).to.eql(false);
+      expect(scope.isMemberOfRunningQuest(user._id, runningQuestParty)).to.eql(false);
     });
 
-    it('does not return true if guild is not included in myGuilds call', function(){
+    it('should return true/false depending on active flag and the method called', function() {
+      expect(scope.isMemberOfPendingQuest(user._id, runningQuestParty)).to.eql(false);
+      expect(scope.isMemberOfPendingQuest(user._id, pendingQuestParty)).to.eql(true);
+      expect(scope.isMemberOfRunningQuest(user._id, runningQuestParty)).to.eql(true);
+      expect(scope.isMemberOfRunningQuest(user._id, pendingQuestParty)).to.eql(false);
+    });
+
+    it('should return false if my user isn\'t part of the quest members', function() {
+      pendingQuestParty.quest.members = {};
+      runningQuestParty.quest.members = {};
+      expect(scope.isMemberOfPendingQuest(user._id, pendingQuestParty)).to.eql(false);
+      expect(scope.isMemberOfRunningQuest(user._id, runningQuestParty)).to.eql(false);
+    });
+  });
+   
+
+  describe("Group Edit Behavior", function() {
```

Comment:
This is describing the `editGroup` method, so it should be `describe('editGroup'`, function () {`